x-pack/test/case_api_integration/common/lib/utils.ts

```diff
@@ -38,34 +68,3 @@ export const deleteConfiguration = async (es: any): Promise<void> => {
     body: {},
   });
 };
-
-export const getConnector = () => ({
```

Comment:
This is not used by tests?

Reply:
nope, i think cuz you had an issue with the `get_connectors` test??